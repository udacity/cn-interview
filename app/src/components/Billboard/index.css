.billboard {
  padding: 1.5em 1em;
  background: #1e8fc1;
  color: white;
}

.billboard-title {
  font-weight: 300;
  font-size: 32px;
}

.billboard-body {
  font-weight: 100;
  font-size: 18px;
}