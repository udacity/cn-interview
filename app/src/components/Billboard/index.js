import React, { Component } from 'react';
import PropTypes from 'prop-types';
import './index.css';

class Billboard extends Component {
  static propTypes = {
    title: PropTypes.string,
    body: PropTypes.string,
  };

  render() {
    const { title, body } = this.props;
    return (
      <div className="billboard">
        <h1 className="billboard-title">{title}</h1>
        <p className="billboard-body">{body}</p>
      </div>
    );
  }
}

export default Billboard;
