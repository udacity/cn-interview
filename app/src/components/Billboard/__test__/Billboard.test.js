import React from 'react';
import { render } from 'enzyme';
import Billboard from '..';

it('renders correctly', () => {
  const wrapper = render(
    <Billboard title="The Title" body="The Body" />
  );
  expect(wrapper).toMatchSnapshot();
});
