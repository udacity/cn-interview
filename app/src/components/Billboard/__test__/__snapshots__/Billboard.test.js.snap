// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="billboard"
>
  <h1
    class="billboard-title"
  >
    The Title
  </h1>
  <p
    class="billboard-body"
  >
    The Body
  </p>
</div>
`;
